src/components/EmptyState/EmptyStateHelp.js

```diff
@@ -0,0 +1,23 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * EmptyStateHelp renders contents of the element
+ */
+const EmptyStateHelp = ({ children, className, ...props }) => {
+  const classes = ClassNames('blank-state-pf-helpLink', className);
```

Comment:
I'm going to call these out, it appears as a slight mistake... in Angular Pf there are a few of these labeled ```blank-STATE-``` instead of ```blank-SLATE-```. We had originally mirrored that syntax, now changing to match ```blank-SLATE-``` since the classes appear made up, probably for JS access.